src/Validation/FormValidation.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validation;
+
+/**
+ * Class to do validation for form submition before sending it to Salesforce
+ * @package App\Validation
+ */
+class FormValidation{
+
+    public static function validationName($name){
+
+        $returnArray  = [
+            'errors' => [],
+            'link' => '#name',
+        ];
+
```

Comment:
I think you need to add an additional wrapper condition here:  e.g 
` if ($phone != null) {
            if (empty($phone)) {
                $returnArray['errors'] = ['Enter a telephone number'];
            } elseif (strlen($phone) > 20) {
                $returnArray['errors'] = ['Telephone number must be 20 characters or fewer'];
            }
        }`

The reason for this is on testing on one of the marketing pages, in the error summary, I'm getting an error for 'telephone number' and 'enter more detail' when there is no such field on the form. 

e.g on this page http://local.crowncommercial.gov.uk/products-and-services/corporate/doc-man-logistics/courier-services